custom_dc/build_image.md

```diff
@@ -124,9 +124,9 @@ If you want to create a new source directory and start from scratch, clone the r
 Run the following command to build the repo:
 
 <pre>
-docker build --tag datacommons-website-compose:<var>DOCKER_TAG</var> \
+docker build --tag datacommons-services:<var>DOCKER_TAG</var> \
 -f build/web_compose/Dockerfile \
```

Comment:
This needs to be `-f build/cdc_services/Dockerfile \`

Reply:
Done.